vars/main.yml

```diff
@@ -9,13 +9,22 @@ postgresql_external:
       rhel-7-x86_64/pgdg-redhat95-9.5-3.noarch.rpm"
     "9.6": "https://download.postgresql.org/pub/repos/yum/9.6/redhat/\
       rhel-7-x86_64/pgdg-redhat96-9.6-3.noarch.rpm"
+    "10": "https://download.postgresql.org/pub/repos/yum/10/redhat/\
+      rhel-7-x86_64/pgdg-redhat10-10-2.noarch.rpm"
   basename:
     "9.4": postgresql94
     "9.5": postgresql95
     "9.6": postgresql96
+    "10": postgresql10
+  setup:
```

Comment:
I hadn't realized that they introduced a variation in the naming scheme. If https://github.com/openmicroscopy/ansible-role-postgresql/pull/14/files#diff-2444ad0870f91f17ca6c2a5e96b26823R44 is the only usage, maybe we want this new dictionary to reflect the full command name i.e. `postgresql94-setup`, `postgresql-10-setup` rather just the basename of the setup command.